web-ui/src/pages/ReceivedRequestsPage.jsx

```diff
@@ -264,16 +256,14 @@ const ReceivedRequestsPage = () => {
       </div>
       <div className="request-section-header">
         <Typography variant="h5">Received Requests</Typography>
-        <IconButton
+        <ExpandMore
+          expand={receivedRequestsExpanded}
           onClick={() => setReceivedRequestsExpanded(!receivedRequestsExpanded)}
+          aria-expanded={receivedRequestsExpanded}
           aria-label="show more"
-          className={
-            receivedRequestsExpanded ? classes.expandOpen : classes.expandClose
-          }
-          size="large"
         >
           <ExpandMoreIcon />
```

Comment:
Why doesn't the `ExpandMore` component supply `ExpandMoreIcon` so it isn't needed in this file?